src/triggers/index.ts

```diff
@@ -0,0 +1,17 @@
+import type BudgetExample from "./budget.json";
+import type CloudLoggingExample from "./cloudLogging.json";
+import type ContainerRegistryExample from "./containerRegistry.json";
+import type DataStreamsExample from "./dataStreams.json";
+import type IotCoreExample from "./iotCore.json";
+import type MessageQueueExample from "./messageQueue.json";
+import type ObjectStorageExample from "./objectStorage.json";
+import type TimerExample from "./timer.json";
+
+export type Timer = typeof TimerExample;
```

Comment:
This is very interesting way to infer types from JSON automatically, but I would ask to define TypeScript interfaces explicitly, since it is more flexible and easier to maintain in future.

BTW, let's add to all type `...Event` suffix so it will be clear for consumer what particular type is intended for.